.github/mergify.yml

```diff
@@ -0,0 +1,22 @@
+queue_rules:
+  - name: default
+    conditions:
+      - check-success=build
```

Comment:
Let's depend on `all`
https://github.com/paritytech/substrate-connect/pull/774

Reply:
That was my first approach but we have only 1 for now, and when a second is added, then this can be added... But sure

Reply:
We already have three: `build (14.x)`, `build (16.x)` and `build (17.x)`